### ISSUE TITLE
Create & publish on only one locale is not working

### ISSUE THREAD

**Issue by Rheaparks** [payloadcms/payload#9704]
### Describe the Bug

I passed to Payload to been able to publish on only one locale (and later on others) but:

1. When I create a post, it create a line on both languages (French & English). Moreover EN is the main language. And if I publish on English, the French version show an empty line in the list (since there is no title for the French locale).
![Image](https://github.com/user-attachments/assets/122dc0bb-f0da-4d94-964e-49dc88d5ba9a)

2. When I click on "Publish in English", the content stay in "Draft". It's working only when I click on "Publish" and so the content is published on every locale.

### Link to the code that reproduces this issue

https://github.com/Rheaparks/payload-debug-locales

### Reproduction Steps

- Complete fields for 1 post (here thingstodo)
- Try to publish on English only.
- Content automatically appear also in French (draft).
- The English content stay to Draft mode.
- Publish globally and both locale will be published.

### Which area(s) are affected? (Select all that apply)

db-vercel-postgres

### Environment Info

```text
Payload: 
Node
Next.js
Vercel Postgres
```

**Comment by maximseshuk** [comment]
I have the same problem. I want to be able to publish in just one specific language. Otherwise, it loses its purpose because it's available in both languages anyway, even if the translation isn't finished.

**Comment by mikaelbalin** [comment]
I have a few issues, but I can't guarantee they're related to localization:  
1. When creating an item in the collection, an additional empty one is created.  
2. If I specify a relationship field, besides the previous issue, the current item becomes impossible to publish. If I remove the relationship field, it becomes editable again.

https://github.com/user-attachments/assets/b708889c-75fa-4009-b499-62250a51b07b

**Comment by mikaelbalin** [comment]
Yes, I checked again, and as soon as I added the single field localization in a separate block, the publishing stopped working, but only when related categories are present.

**Comment by mikaelbalin** [comment]
So, here are the steps to reproduce:  

1. Create a new project with the **Website** template and choose **PostgreSQL** as the database.  
2. Add the following localization config:  

   ```javascript
   localization: {
       defaultLocale: 'en',
       locales: ['en', 'fr'],
   },
   ```  

3. In the UI, create a category.  
4. Create the first post, attach the category to it, and publish.  
5. Create the second post, attach the same category, and try to publish.  

### **Current Result:**  
❌ Can't publish the new version.  @

### **Expected Result:**  
✅ Should successfully publish the new version.  

I'm ready to investigate this issue and would appreciate any suggestions on where to start. @

**Comment by mikaelbalin** [comment]
So, it turns out, that a gave the relation the same name as `category` collection.

**Comment by nezzard** [comment]
> So, it turns out that I gave the relation the same name as the category collection.

Any solutions?

**Comment by mikaelbalin** [comment]
@nezzard check the relation names, they should be different from slug names

<img width="984" height="462" alt="Image" src="https://github.com/user-attachments/assets/c84ce9fd-852f-49db-9340-7dc0715309d8" />

**Comment by JessRynkar** [comment]
Hi all, there were some limitations with the existing functionality for publishing individual locales. We just released an experimental feature `localizeStatus` in [v3.72.0](https://github.com/payloadcms/payload/releases/tag/v3.72.0), this maintains an individual status per locale and generally improves the process. If possible, I would try out this new feature and see if your issues are resolved.

If you are still having issues, please open a new issue as this one is over a year old and doesn't state which version is affected.
